Explorer/Assets/DCL/SDKComponents/Tween/Components/SDKTweenComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Arch.Core;
+using DCL.ECSComponents;
+using DG.Tweening;
+using UnityEngine;
+
+namespace ECS.Unity.Tween.Components
+{
+    public struct SDKTweenComponent
+    {
+        public Entity globalWorldEntity;
+
+        public bool IsDirty { get; set; }
+        public bool removed;
+        public bool isPlaying;
+        public float CurrentTime;
+        public Tweener tweener;
+        public PBTween.ModeOneofCase tweenMode;
+        public PBTween CurrentTweenModel;
```

Comment:
You should not store the `PB` component directly: it is pooled and reused internally and it's a reference type so you have a chance to end-up with completely irrelevant data

Reply:
okay, created a model that stores the PBTween data inside the `SDKTweenComponent` + made the component poolable using `IPoolableComponentProvider `as I noticed it's the way its used in other systems, so the "main" `SDKTweenComponent ` can be reused when PBtween is removed from entities. Please let me know if its a good use of it or not at all xD